using-valtimo/access-control/configuring-roles.md

```diff
@@ -0,0 +1,47 @@
+# Configuring roles
+In order to allow users to perform certain actions, permissions have to be set up. Before permissions, however,
+roles should be created that are also contained in a user JWT token. For example, if a user has `ROLE_USER` in Keycloak.
+In order to create permissions that apply to that specific role, the role has to first be known by Valtimo. Keep in mind
+`ROLE_ADMIN` will always have access to items in the admin menu.
+
+For information on how to configure permissions, see [this page](configuring-permissions.md).
+
+## Creating a role
+Creating a role can be done by going to the `Admin` menu. There, under `A&E` you will find `Access Control`. This will
+show the list of roles that have been configured. In order to create a new role, make sure no other roles are selected,
+and press `Add new role`. A dialog will show that allows you to specify the name of the role before creating it.
+
+![creating-a-role-example](img/creating-a-role.png)
+
+## Editing a role
+Roles can be edited by clicking on that role. Then, by clicking on the three dots next to the `Save` button, the role
+can then be updated.
+
+![updating-a-role-example](img/updating-a-role.png)
+
+## Deleting a role
+Roles can be deleted by clicking on that role. Then, by clicking on the three dots next to the `Save` button, the role
+can then be deleted.
+
```

Comment:
Maybe explain what happens to permissions when deleting a role?

EDIT:
Oh wait, you have that in a paragraph below. Can you combine these?

Reply:
done